autoload/ale/handlers/languagetool.vim

```diff
@@ -2,18 +2,26 @@
 " Description: languagetool for markdown files
 "
 call ale#Set('languagetool_executable', 'languagetool')
+call ale#Set('languagetool_commandline_jar', 'languagetool-commandline.jar')
```

Comment:
Make this an empty string by default, or otherwise determine a valid path to the .jar.

Reply:
make it empty by default.